.github/ISSUE_TEMPLATE/bug_form.yaml

```diff
@@ -0,0 +1,92 @@
+name: Bug Report Form
+description: Report a bug or a similiar issue.
+labels:
+  - needs:triage
+title: "Bug: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue. Please fill out the below template with as much detail as possible. Incomplete bug reports are likely to be closed without comment.
+
+        If you want to suggest a feature use the
+        [Feature Request template](https://github.com/akaihola/darglint2/issues/new?template=feature_request.md)
+        instead.
+
+  - type: checkboxes
+    id: checklist-existing-issues
+    attributes:
+      label: Existing Issues
+      description: |
+        Please  do not create a new issue for a problem that was already reported but use the existing thread.
+        However issues that only exist in the original [darglint](https://github.com/terrencepreilly/darglint)
+        repository may be reported again in this one.
+        Please link all issues that might relate to this report below.
+      options:
+        - label: |
+            I have searched the [darglint2 issue tracker](https://github.com/akaihola/darglint2/issues?q=is%3Aissue)
+            for open and closed issues to avoid duplicates.
+          required: true
+        - label: |
+            I have searched the [darglint issue tracker](https://github.com/terrencepreilly/darglint/issues?q=is%3Aissue)
+            for open and closed issues.
+          required: true
```

Comment:
Why would we require people to search old Darglint issues? I think that's a bit of a burden and I can't immediately see the benefit.

Reply:
I was thinking that for the some issues the solution would already have been found in the old repository. Also there might be valuable information in old issues reporting the same bug. Including this in the templates, gives each reporter a bit more work to do but reduces the work we have in the end. However I am ok with dropping it if you think it doesn't provide that much benefit.

Reply:
...or at least not requiring it?

Reply:
That's what I will go with for now.